plugins/woocommerce/includes/class-wc-ajax.php

```diff
@@ -298,7 +298,12 @@ private static function update_order_review_expired() {
 				'fragments' => apply_filters(
 					'woocommerce_update_order_review_fragments',
 					array(
-						'form.woocommerce-checkout' => '<div class="woocommerce-error">' . __( 'Sorry, your session has expired.', 'woocommerce' ) . ' <a href="' . esc_url( wc_get_page_permalink( 'shop' ) ) . '" class="wc-backward">' . __( 'Return to shop', 'woocommerce' ) . '</a></div>',
+						'form.woocommerce-checkout' => wc_print_notice(
+							esc_html__( 'Sorry, your session has expired.', 'woocommerce' ) . ' <a href="' . esc_url( wc_get_page_permalink( 'shop' ) ) . '" class="wc-backward">' . __( 'Return to shop', 'woocommerce' ) . '</a>',
```

Comment:
You changed "Sorry, your session has expired" to `esc_html__` but not "Return to shop" - any reason why one changed not the other?

Reply:
I missed it - there was no escape before. I'll add it.